pgrx/src/datum/array.rs

```diff
@@ -170,28 +145,61 @@ impl<'mcx, T: UnboxDatum> Array<'mcx, T> {
         ArrayIterator { array: self, curr: 0, ptr }
     }
 
+    /// Return an iterator of `Nullable<T>`
+    /// Behavior is identical to ArrayIterator, but with `Nullable<T>`'s,
+    /// syntax, meaning `next()` returns `Option<Nullable<T>>`
+    /// instead of `Option<Option<T>>`.
+    pub fn iter_nullable(&self) -> NullableArrayIterator<'_, T> {
+        NullableArrayIterator { inner: self.iter() }
+    }
```

Comment:
I would rather there be no new public functions on Array.

Reply:
Resolved in 8bc1648